web/viewer.css

```diff
@@ -209,6 +209,13 @@
   }
 }
 
+.viewer {
```

Comment:
First of all, this class-name doesn't actually exist. Secondly, these new rules *must* be placed in `web/pdf_viewer.css` instead since otherwise the standalone viewer-components will not work.